docs/ref/forms/fields.txt

```diff
@@ -1222,6 +1226,13 @@ method::
             ...
             </select>
 
+    ``ModelChoiceField`` also has the attribute:
```

Comment:
I'd drop this sentence, and then just have the `.. attribute` that follows.

Reply:
I'm fine dropping it, but I'm concerned that without it, `iterator` looks like it lives below the heading "ModelChoiceField also takes two optional arguments:" and so might be misinterpeted as an argument. WDYT?